src/plan_ordered_append.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#include <postgres.h>
+#include <nodes/relation.h>
+#include <nodes/value.h>
+#include <optimizer/paths.h>
+#include <optimizer/pathnode.h>
+#include <optimizer/tlist.h>
+#include <utils/builtins.h>
+#include <utils/typcache.h>
+
+#include "compat.h"
+#include "guc.h"
+#include "hypertable.h"
+#include "hypertable_restrict_info.h"
+#include "constraint_aware_append.h"
+#include "planner.h"
+#include "plan_ordered_append.h"
+
+/*
+ * Check if conditions for doing ordered append optimization are fulfilled
+ */
+bool
+ts_ordered_append_do_optimization(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht, bool *reverse)
```

Comment:
Function name made me think that this method does the actual optimization. I'd recommend something more intuitive like `ts_ordered_append_can_optimize` or similar

Reply:
changed